### PR TITLE
feat: Arbitrum x EigenDA batch poster usage overview

### PR DESCRIPTION
**Pull request by epociask**

### CODE REVIEW

File: docs/integrations-guides/rollup-guides/orbit/deployment.md [hunk 1]
```diff
@@ -34,7 +36,11 @@ The script will take a few minutes to complete as it prints out the addresses of
 
 ## How to deploy a Rollup on Testnet using our UI
 
-While you can interact with the deployed Rollup creator directly, we recommend using the [EigenDA x Orbit chain deployment portal](https://orbit.eigenda.xyz/) to deploy a rollup for a friendly UX and easy-to-use configs. Currently, the only supported testnets are Ethereum Holesky and Arbitrum Sepolia.
+While you can interact with the deployed Rollup creator directly, we recommend using our [chain deployment portal](https://orbit.eigenda.xyz/) to deploy a rollup for a friendlier devx and easy-to-use configs. Currently, the only supported testnets are:
```
Comment (mmurrs) [inline docs/integrations-guides/rollup-guides/orbit/deployment.md:39]: Is the chain deployment portal something we are scoping to support for Optimism as well?

Initial thought is to specify Orbit x EigenDA

File: docs/integrations-guides/rollup-guides/orbit/overview.md [hunk 2]
```diff
@@ -3,23 +3,25 @@ title: Technical overview
...
-### Runbook
+## Runbook
 
 Core Arbitrum is a highly complicated composition of many software repositories and programming languages. To better demystify key system flows, we've developed an operational developer [runbook](https://eigen-labs.notion.site/Arbitrum-x-EigenDA-Developer-Runbook-12466062c1a7495ebc1d803169c37644?pvs=4) which describes core testing and system procedures.
 
-## ETH Layer2 vs Layer3 deployments
+## ETH L2 vs L3 deployments
 
-Layer3s using Arbitrum with EigenDA are a M0 integration unlike Layer2s which are M1. This means both degraded security and throughput when currently using L3s with EigenDA. Please advise our Integrations [Overview](../integrations-overview.md) for a more comprehensive overview of different EigenDA rollup stages.
+L2s using Arbitrum with EigenDA are a M0 integration unlike L3s which are M1. This means both degraded security and throughput when currently using L3s with EigenDA. Please advise our Integrations [Overview](../integrations-overview.md) for a more comprehensive overview of different EigenDA rollup stages.
```
Comment (mmurrs) [inline docs/integrations-guides/rollup-guides/orbit/overview.md:14]: nice

File: docs/integrations-guides/rollup-guides/orbit/overview.md [hunk 3]
```diff
@@ -34,6 +36,42 @@ For replicating EigenDA interactions in a local testing environment, we utilize
 
 [EigenDA Proxy](https://github.com/Layr-Labs/eigenda-proxy) is used for secure and optimized communication between the rollup and the EigenDA disperser. Arbitrum uses the [*Simple Commitment Mode*](https://github.com/Layr-Labs/eigenda-proxy?tab=readme-ov-file#simple-commitment-mode) for client/server interaction and representing DA certificates. Read more about EigenDA Proxy and its respective security features [here](./../../dispersal/clients/eigenda-proxy.md).
 
+### Posting batches to EigenDA
+
+Please ensure that changes made to batch poster configs are globally applied across all your batch poster instances. If not, there could be inconsistencies that arise due to deviations in collective processing logic. To learn more about the Arbitrum batch poster please advise our following overview [spec](https://hackmd.io/@epociask/ByHk6x_TC).
+
+**Adjusting maximum batch size**
+
+Currently, the batch poster defaults to a maximum of `16mib` when dispersing batches to EigenDA. This can be adjusted to a lower threshold directly within the batch poster section of your node config:
```
Comment (mmurrs) [inline docs/integrations-guides/rollup-guides/orbit/overview.md:45]: *16MB

File: docs/integrations-guides/rollup-guides/orbit/overview.md [hunk 4]
```diff
@@ -34,6 +36,42 @@ For replicating EigenDA interactions in a local testing environment, we utilize
...
+            ...
+            "max-eigenda-batch-size": 12_000_000, // 12 MB
+        }
+    }
+```
+
+
+**Enabling Failover**
+
+To remove a trust assumption on the liveness of EigenDA for the liveness of the rollup, we've extended the Arbitrum Nitro batch poster's logic to support opt-in failover to other DA destinations (e.g, AnyTrust, 4844, calldata) in the event of indicated service unavailability from EigenDA. This logic is disabled by default but can be enabled by making the following update to your batch poster config with the following field:
```
Comment (mmurrs) [inline docs/integrations-guides/rollup-guides/orbit/overview.md:61]: EIP-4844

File: docs/integrations-guides/rollup-guides/orbit/overview.md [hunk 5]
```diff
@@ -34,6 +36,42 @@ For replicating EigenDA interactions in a local testing environment, we utilize
...
+            "enable": true,
+            ...
+            "enable-eigenda-failover": true, 
+        }
+    }
+```
+
+To learn more about our Arbitrum failover design methodology, please advise the following [spec](https://hackmd.io/@epociask/SJUyIZlZkx).
+
 # Diff Overview
```
Comment (mmurrs) [inline docs/integrations-guides/rollup-guides/orbit/overview.md:75]: What's the diff specifically referencing? Is this our forked Arbitrum Orbit Nitro Fork? 

If so can we be explicit?

File: docs/integrations-guides/rollup-guides/orbit/overview.md [hunk 6]
```diff
@@ -44,6 +82,7 @@ Many modifications were made to successfully support fraud proofs and secure bat
 
 - [Extended](https://github.com/Layr-Labs/nitro/blob/e8981ff2f09720b6627e751d8bd3146277c7a01b/arbnode/batch_poster.go#L113) batch poster to take in an `eigenDAWriter` struct that writes blobs to DA via eigenda-proxy
     - [Embed](https://github.com/Layr-Labs/nitro/blob/e8981ff2f09720b6627e751d8bd3146277c7a01b/arbnode/batch_poster.go#L1030-L1075) ABI calldata for tx submissions to `SequencerInbox`
+    - [Provide](https://github.com/Layr-Labs/nitro/pull/46) programmatic failover feature to use other DA destinations when EigenDA is unavailable.
 - [Extended](https://github.com/Layr-Labs/nitro/blob/e8981ff2f09720b6627e751d8bd3146277c7a01b/arbnode/sequencer_inbox.go#L174-L211) inbox message derivation to support type processing for an eigenda batch type (i.e, `0xed` prefix)
```
Comment (mmurrs) [inline docs/integrations-guides/rollup-guides/orbit/overview.md:86]: *EigenDA

File: docs/integrations-guides/rollup-guides/orbit/overview.md (hunk 3 above)
Comment (epociask) [reply docs/integrations-guides/rollup-guides/orbit/overview.md:45]: why would it be mb and not mib?

File: docs/integrations-guides/rollup-guides/orbit/overview.md (hunk 5 above)
Comment (epociask) [reply docs/integrations-guides/rollup-guides/orbit/overview.md:75]: the below diff demarcates between between each specific fork

File: docs/integrations-guides/rollup-guides/orbit/overview.md (hunk 6 above)
Comment (epociask) [reply docs/integrations-guides/rollup-guides/orbit/overview.md:86]: ` programmatic failover feature to use other DA destinations when EigenDA is unavailable.` ?

File: docs/integrations-guides/rollup-guides/orbit/deployment.md (hunk 1 above)
Comment (epociask) [reply docs/integrations-guides/rollup-guides/orbit/deployment.md]: [4b309db](https://github.com/Layr-Labs/eigenda-docs/pull/37/commits/4b309db8d48325e0051f222632aad8fea8a70d45)

**Review by mmurrs — APPROVED** [review]
Minor updates before merging left in open comments

**Pull request merged**